src/alire/alire-utils-user_input.adb

```diff
@@ -115,6 +126,19 @@ package body Alire.Utils.User_Input is
                end loop;
             end if;
 
+            --- Check if the user input the whole answer
+            if Input'Length >= 1 then
```

Comment:
I think you can remove this check, does not seem necessary to me.

Reply:
I agree and removed it.